boards/samr21-xpro/Makefile.include

```diff
@@ -5,4 +5,16 @@ export CPU_MODEL = samr21g18a
 # set edbg device type
 EDBG_DEVICE_TYPE = atmel_cm0p
 
+# export ROM partitions
+export ROM_SIZE = 0x40000
+export SLOT0_ADDR = 0x0
+export SLOT0_SIZE = 0x4000
+CFLAGS +=-DSLOT0_SIZE=$(SLOT0_SIZE)
```

Comment:
Yes, maybe we can add them to the exported variables in the main makefile intended for this.